relay/kusama/src/lib.rs

```diff
@@ -317,10 +328,11 @@ impl pallet_balances::Config for Runtime {
 	type MaxReserves = MaxReserves;
 	type ReserveIdentifier = [u8; 8];
 	type WeightInfo = weights::pallet_balances::WeightInfo<Runtime>;
-	type FreezeIdentifier = ();
-	type MaxFreezes = ();
+	type FreezeIdentifier = RuntimeFreezeReason;
+	type MaxFreezes = ConstU32<1>;
```

Comment:
I think we should probably increase this one, and also the max holds.  
Maybe to 8 or 16.

Reply:
Increased to 8 for Polkadot and Kusama. There's also 2nd instance of balances pallet at Kusama. Do you think it also needs to be increased?